README.md

```diff
@@ -1,31 +1,2434 @@
-The prototype permitting application for the City of Boston  
-
-## Issue
-Moving day--September 1--is a big day in Boston. 
-A large number of permits for moving trucks and moving pods are pulled for the day. 
-While moving occurs citywide, permits are always pulled for the neighborhoods of Fenway, Mission Hill, and the North End. 
-These neighborhoods are small, and permit spaces are finite.
-These neighborhoods host a high density of movers.
-Permits are presently issued for entire days, but trucks or pods often occupy spaces for a set of hours.
-Only some people are able to hold moving spaces legally, producing both free riding on permitted spaces as well as illegal moves which may cause further congestion and safety hazards.
-
-## Questions
-Focusing on one neighborhood with high moving permitting, can we create a system that maximizes people’s ability to move legally and safely?
-For what streets are moving permits ALWAYS pulled on moving day (Sept 1)?
-How would people ‘reserve’ a space?
-How long should people ‘reserve’ a space for?
-How would a permitting officer/ticketing officer know who has the space reserved on a specific day and time?
-
-## Scope of Work
-Gordon College Consultants (below) will help Principle Stakeholders identify preliminary answers to the above Questions by prototyping a digital solution to the above Issue. The prototyped digital solution will require a deep spatial understanding of permitting data, a deep understanding of the experience of the current permitting system, and the skills and resources necessary to fabricate a working alpha (or beta!) digital system. This prototype will be delivered by the end of the Gordon College spring semester. Should the Gordon College team submit with ample time left (and should they be interested in pursuing a next step of those prototype), there is possibility for additional scoping.
-
-The New Urban Mechanics will provide access to data, key stakeholders, and other resources as necessary. They will also provide regular feedback (frequency established in partnership with the team).
-
-The prototype delivered by Gordon College Consultants will be used as a proof-of-concept design and catalyst for internal stakeholder conversations around moving day permitting, and may not be adopted as a final solution to the above Issue.
-
-### Moving Truck Permit Dataset - https://data.boston.gov/dataset/open-moving-truck-permits
-### Current Street Sweeping Schedules - https://data.boston.gov/dataset/street-sweeping-schedules
-### Current Moving Permit Reservation System - https://www.boston.gov/how-reserve-parking-spot-your-moving-truck
-### General Moving Day Information - https://www.boston.gov/movingx
-### Proposal- https://docs.google.com/document/d/1U_7Pi0U9DdZ-eEz-trZTXDP3XaLs8rmhaDX_kh44vkM/edit#heading=h.6cxtcn9dvq82
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
```

Comment:
Did you mean to overwrite your README.md with this generated one?  Having both, with different names, seems logical.